src/main/java/uk/gov/hmcts/reform/data/ingestion/camel/service/dto/Email.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.data.ingestion.camel.service.dto;
+
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.data.ingestion.camel.exception.EmailFailureException;
+
+import java.util.List;
+
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+
+@Data
+@Builder
+public class Email {
+
+    private String from;
+
+    private String subject;
+
+    private List<String> to;
+
+    private String messageBody;
+
+    public void validate() {
+        if (isNull(from) || isEmpty(to) || isNull(subject)) {
+            throw new EmailFailureException("Can't send email as some of the mandatory email parameters are missing - "
+                    + " From: {" + from + "} To: {" + to + "} Subject: {" + subject + "}.");
```

Comment:
I might be wrong here - do you think it is safe to have pii data (from/to) in the exception message?

Reply:
Don't think there is any sensitive data here so I have included them in exception message. Let me know if you think otherwise.

@shreedhar-hmcts @souravroy2021 your inputs are welcome here.

Reply:
@shreedhar-hmcts @souravroy2021 @SujeethKumarAri Since we haven't reached a consensus on this, I am merging this PR for now. If this has to be changed, let's handle it as part of another ticket. This is blocking the JRD data load PR.